source/ht_application.cpp

```diff
@@ -98,6 +100,8 @@ namespace Hatchit {
                 wparams.y = -1;
                 wparams.width = 800;
                 wparams.height = 600;
+                wparams.centerX = 400;
```

Comment:
Shouldn't CenterX and CenterY be calculated at runtime rather than specified in parameters?